tools/build_framework.sh

```diff
@@ -34,6 +66,23 @@ _notify_github() {
     GIT_REPOSITORY_ROOT=$REPO_ROOT_DIR ${TOOLS_DIR}/github_update.py $1 build:${FRAMEWORK_NAME} $2
 }
 
+# Used below in-order, but here for cli-only
+build_cli() {
+    # CLI (Go):
+    # /home/user/dcos-commons/frameworks/helloworld/cli => frameworks/helloworld/cli
+    REPO_CLI_RELATIVE_PATH="$(echo $CLI_DIR | cut -c $((2 + ${#REPO_ROOT_DIR}))-)"
+    ${TOOLS_DIR}/build_cli.sh ${CLI_EXE_NAME} ${CLI_DIR} ${REPO_CLI_RELATIVE_PATH}
+    if [ $? -ne 0 ]; then
+        _notify_github failure "CLI build failed"
```

Comment:
Which script, build_framework, or _notify_github? If the latter, I must have not updated master before createing this branch.  Checking now.